src/data/script/ConstraintChangerModPlugin.java

```diff
@@ -46,14 +48,88 @@ public class ConstraintChangerModPlugin extends BaseModPlugin {
+    public static final String FIELD_BULK_TRANSPORT_CARGO_CAPACITY_THRESHOLD = "constraintchanger_skill_industry_BulkTransport_CargoCapacityThreshold";
+    public static final String FIELD_BULK_TRANSPORT_FUEL_CAPACITY_MAX_PERCENTAGE = "constraintchanger_skill_industry_BulkTransport_FuelCapacityMaxPercent";
+    public static final String FIELD_BULK_TRANSPORT_FUEL_CAPACITY_THRESHOLD = "constraintchanger_skill_industry_BulkTransport_FuelCapacityThreshold";
+    public static final String FIELD_BULK_TRANSPORT_PERSONNEL_CAPACITY_MAX_PERCENTAGE = "constraintchanger_skill_industry_BulkTransport_PersonnelCapacityMaxPercent";
+    public static final String FIELD_BULK_TRANSPORT_PERSONNEL_CAPACITY_THRESHOLD = "constraintchanger_skill_industry_BulkTransport_PersonnelCapacityThreshold";
+    public static final String FIELD_BULK_TRANSPORT_BURN_BONUS = "constraintchanger_skill_industry_BulkTransport_BurnBonus";
+
+    public static final String FIELD_FIELD_REPAIRS_MIN_HULL = "constraintchanger_skill_industry_FieldRepairs_MinHull";
+    public static final String FIELD_FIELD_REPAIRS_MAX_HULL = "constraintchanger_skill_industry_FieldRepairs_MaxHull";
+    public static final String FIELD_FIELD_REPAIRS_MIN_CR = "constraintchanger_skill_industry_FieldRepairs_MinCR";
+    public static final String FIELD_FIELD_REPAIRS_MAX_CR = "constraintchanger_skill_industry_FieldRepairs_MaxCR";
+    public static final String FIELD_FIELD_REPAIRS_REPAIR_RATE_BONUS = "constraintchanger_skill_industry_FieldRepairs_RepairRateBonus";
+    public static final String FIELD_FIELD_REPAIRS_INSTA_REPAIR_PERCENT = "constraintchanger_skill_industry_FieldRepairs_InstaRepairPercent";
+
+    public static final String FIELD_CONTAINMENT_PROCEDURES_CREW_LOSS_REDUCTION = "constraintchanger_skill_industry_ContainmentProcedures_CrewLossReduction";
+    public static final String FIELD_CONTAINMENT_PROCEDURES_FUEL_SALVAGE_BONUS = "constraintchanger_skill_industry_ContainmentProcedures_FuelSalvageBonus";
+    public static final String FIELD_CONTAINMENT_PROCEDURES_FUEL_USE_REDUCTION_MAX_PERCENT = "constraintchanger_skill_industry_ContainmentProcedures_FuelUseReductionMaxPercent";
+    public static final String FIELD_CONTAINMENT_PROCEDURES_FUEL_USE_REDUCTION_MAX_FUEL = "constraintchanger_skill_industry_ContainmentProcedures_FuelUseReductionMaxFuel";
+
+    public static final String FIELD_DERELICT_OPERATIONS_DP_COST_REDUCTION_PER_DMOD = "constraintchanger_skill_industry_DerelictOperations_DPCostReductionPerDmod";
+
+    /************************
+     * SHARED SKILLS FIELDS *
+     ************************/
+
+    public static final String FIELD_SHARED_CREW_TRAINING_AND_FLUX_REGULATION_OP_THRESHOLD = "constraintchanger_skill_shared_OPThreshold";
+    public static final String FIELD_SHARED_FIGHTER_BAYS = "constraintchanger_skill_shared_FighterBays";
+    public static final String FIELD_SHARED_FIELD_REPAIRS_AND_CONTAINMENT_PROCEDURES_OP_THRESHOLD = "constraintchanger_skill_shared_OPThreshold2";
```

Comment:
might be a decent idea to extract all of these to some new class, instead of them being here...
`LunaFields` perhaps? `FieldNameConstants` ? think about it.